config/environments/production.rb

```diff
@@ -65,33 +64,25 @@
   # Send deprecation notices to registered listeners.
   config.active_support.deprecation = :notify
 
+  # Log disallowed deprecations.
+  config.active_support.disallowed_deprecation = :log
+
+  # Tell Active Support which deprecation messages to disallow.
+  config.active_support.disallowed_deprecation_warnings = []
+
   # Use default logging formatter so that PID and timestamp are not suppressed.
   config.log_formatter = ::Logger::Formatter.new
 
   # Use a different logger for distributed setups.
-  # require 'syslog/logger'
+  # require "syslog/logger"
   # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
 
   if ENV['RAILS_LOG_TO_STDOUT'].present?
-    logger = ActiveSupport::Logger.new($stdout)
+    logger           = ActiveSupport::Logger.new($stdout)
     logger.formatter = config.log_formatter
-    config.logger = ActiveSupport::TaggedLogging.new(logger)
+    config.logger    = ActiveSupport::TaggedLogging.new(logger)
   end
 
-  # see https://github.com/shadabahmed/logstasher
-  config.logstasher.enabled = true
-  config.logstasher.log_level = Logger::INFO
-  config.logstasher.logger = config.logger
-  config.logstasher.controller_enabled = true
-  config.logstasher.mailer_enabled = false
-  config.logstasher.record_enabled = false
-  config.logstasher.view_enabled = false
-  config.logstasher.job_enabled = true
-  config.logstasher.suppress_app_log = true
-  config.logstasher.source = "laa-court-data-ui-production-#{ENV.fetch('ENV', nil)}"
-  config.logstasher.backtrace = false
-  config.logstasher.logger_path = 'log/logstasher.log'
```

Comment:
Should these lines have been removed? Looks like they are config for logstasher and not rails related.

Reply:
I've reinstated this as it seems it was removed accidentally